src/whatsnew/3.2.rst

```diff
@@ -57,4 +113,140 @@ Features and Enhancements
+
+* :ghissue:`3600`, :ghissue:`3047`, :ghissue:`3019`: The default ``slack`` channel for
+  smoosh auto-compaction has been increased to a more reasonable value, reducing load
+  on systems that would have normally been idle in CouchDB 2.x (where no auto-compaction
+  daemon exists).
+
+* :ghissue:`3711`: Changes feeds may no longer rewind after shard moves, assumimg the
+  node and range specified by the changes feed nonce can still match an existing node's
+  shard.
+
+Bugfixes
+--------
+
+* Complete retirement of the JavaScript test suite - replaced by Elixir. Hooray!
+* :ghissue:`3165`: Allow configurability of JWT claims that require a value. Also fixes
+  :ghissue:`3232`. Further, :ghissue:`3392` no longer validates claims provided that
+  CouchDB does not require.
+* :ghissue:`3160`, :ghissue:`3161`: The ``run_queue`` statistic now returns valid
+  information even when using Erlang BEAM dirty CPU and IO queues.
+* :ghissue:`3162`: Makefiles updated to include local configs & clean configs when
+  running ``make devclean``.
+* :ghissue:`3195`: The ``max_document_size`` parameter now has a clearer explanation in
+  ``default.ini``.
+* :ghissue:`3207`, :ghissue:`2536`: Improve the ``INSTALL.Unix.md`` file.
+* :ghissue:`3212`: Base and extra headers are properly combined when making replicator
+  requests that contain duplicate headers.
+* :ghissue:`3201`: When using a POST with request body to pass parameters to a view-like
+  request, the boolean parameters are accepting only JSON strings, but not booleans.
+  Now, CouchDB accepts ``true`` and ``false`` for the ``stable`` parameter, in addition
+  to ``"true"`` and ``"false"``.
+  comment in
+* :ghissue:`1988`: Attachment operations ``PUT /db/doc`` and ``POST /db`` now perform
+  consistent attachment name validation.
+* :ghissue:`3249`: Documents with lots of conflicts no longer blow up couchjs if the
+  user calls ``_changes`` with a JS filter and with ``style=all_docs``.
+* :ghissue:`3144`: Respawning compaction jobs to catch up with intervening changes are
+  now handled correctly by the smoosh monitor.
+* :ghissue:`3252`: CouchDB now exports the ``couch_util:json_decode/2`` function to
+  support maps instead of the default data structure.
+* :ghissue:`3255`, :ghissue:`2558`: View files that have incorrect ``db_headers``
+  now reset the index forcing a rebuild.
+* :ghissue:`3271`: Attachments that are stored uncompressed but later replicated to
+  nodes that compress the attachment no longer fail an internal md5 check that would
+  break eventual consistency between nodes.
+* :ghissue:`3277`: ``req_body`` requests that have ``req_body`` set already now
+  properly return the field without parsing.
+* :ghissue:`3279`: Some default headers were missing from some responses in replication,
+  including ``X-CouchDB-Body-Time`` and ``X-Couch-Request-ID``.
+* :ghissue:`3329`, :ghissue:`2962`: CouchDB no longer returns broken couchjs processes
```

Comment:
couch_js?

Reply:
Is it the Erlang process that's broken, or the external process? If the external, the binary is called `couchjs`. If the former, `couch_js.